src/wasm32/ffitarget.h

```diff
@@ -44,7 +44,7 @@ typedef enum ffi_abi {
   FFI_WASM32, // "raw", no structures or varargs
   FFI_WASM32_EMSCRIPTEN, // structures, varargs, and split 64-bit params
```

Comment:
The `split 64-bit params`-phrase should probably be changed as well, since that is not needed when linking with `-sWASM_BIGINT` and building with `-DWASM_BIGINT`.

Reply:
Right. We also have closures, and I don't really care about `WASM32` and have no idea if it works.